html/browsers/offline/application-cache-api/api_update.html

```diff
@@ -11,12 +11,24 @@
     <script>
       var cache = window.applicationCache;
 
-      test(function() {
-        try {
-          cache.update()
-          assert_true(true, "update method test")
-        } catch (e) {
-          assert_unreached("update method failed.");
+      async_test(function(t) {
+        var next = t.step_func(function() {
+          cache.onnoupdate = cache.oncached = null;
+
+          try {
```

Comment:
I thought the same thing. I didn't want to conflate the technical fix with a
more subjective improvement which may have stalled the review. Happy to
simplify that with your support :)